### ISSUE TITLE
install the trailers channel

### ISSUE THREAD

**Issue by tillhanke** [jellyfin/jellyfin-web#4375]
I just stumbled upon the Trailer section and got confused, how to find the "trailer channel". Looking into the jellyfin repo I found a old Issue (https://github.com/jellyfin/jellyfin/issues/2963) in which this is discussed at length. There is also a plugin mentioned:
https://github.com/crobibero/jellyfin-plugin-tmdb-trailers
which will include the trailers into jellyfin.

Now I think it would be better for future users, if this way is not necessary to find the plugin. Couldn't it just be mentioned in the error text:
https://github.com/jellyfin/jellyfin-web/blob/6eb760b03ca438bd7a334e0e930bb777a6671856/src/strings/en-us.json#L1090
Either mention the closed issue, or the plugin directly?

Thanks for the hard work! I really love Jellyfin.

**Comment by Maltavius** [comment]
IMO the Trailer-tab should just list all media that is in a Folder named Trailer in all libraries or ends with -trailer.
Use the Poster and title from the parent item to make it look good.

**Comment by Maltavius** [comment]
> > not everyone is a developer or has the time to spend.
> 
> This also applies to the very few active jellyfin-web contributors. The linked issue was locked because it went offtopic. If someone opens a pull request to modify the message (by either removing it or rephrasing or whatever) I'm confident it will be merged without problem.

Thank you for taking the time to explain why the problem won't be fixed. 

If it was just changing the text in line 1090 of en-us.json I would. But it involves learning how to make a branch, probably downloading the entire source code for Jellyfin, then learning how to make a pull request

"MessageNoTrailersFound": "This tab won't show anything unless you install the trailers plugin found here: https://github.com/crobibero/jellyfin-plugin-tmdb-trailers",

**Comment by Maltavius** [comment]
Just hide the trailer channel until this is fixed.
It's still an issue and has been an issue.

**Comment by kebogaer** [comment]
> Just hide the trailer channel until this is fixed. It's still an issue and has been an issue.

I agree, this is buggy.
Ok admitted when you google you will eventually find the tmdb-trailer plugin, but that;s besides this issue.

This tab is confusing for new users and should either be removed or updated with working funcitonality

**Comment by thornbill** [comment]
PRs are still welcome. Although any direct mentions of the trailers plugin would not be accepted since it may violate terms of service.

**Comment by vreelb** [comment]
For anyone following this issue: I have an open PR (mentioned above) that addresses the tab always being present, hiding it by default unless the backend returns trailers.

I'm posting an update here because it is still waiting for further feedback and my upcoming schedule is such that I may not have time to address those comments when they eventually arrive. I want to make sure that anyone in the future looking at this is aware that I may not be taking the PR above to the finish line, but it is still worth fixing (and my branch can be used as a starting point).

I didn't anticipate this was going to be a 2+ month endeavor when I started it, if I had known I probably wouldn't have bothered. I was doing this to test the waters and frankly this has killed any motivation I had to contribute more to Jellyfin, which is a shame because I like using it.

**Comment by tr1p0p** [comment]
Yeah, i really wonder how such a issue can stay like for this for like, a all year, even when someone put some work to fix that, but for some reasons, it stay on PR for like, months..

Is not that much.

**Comment by vreelb** [comment]
Just closed the PR, what a waste of my time.

**Comment by shandogthecat** [comment]
I just reinstalled Jellyfin for the first time in about a year, with hopes of contibuting this time. Seeing this is all very unfortunate. (I too was confused by the trailers tab.)

**Comment by IlIiIlIIlIiIlIlIIIlI** [comment]
How is someone supposed to get the trailers channel working if running Jellyfin on Synology via community package (not docker)? I can't place files manually into locations like the TMDB github states. Very confusing.

**Comment by benhovinga** [comment]
I can't believe that the Trailers tab is still here. It's problematic existence has been known to the team since at least [2020](https://github.com/jellyfin/jellyfin/issues/2963#issue-605094532).

At this point I would like to see the trailers tab removed outright just to end the confusion. @vreelb has done the work to make it conditionally visible (twice), which is much better than leaving it for a _potential_ plugin (that doesn't exist in the standard plugin repository) to _maybe_ populate the tab in the future. I understand that an [external plugin](https://github.com/crobibero/jellyfin-plugin-tmdb-trailers) exists that is able to populate this tab, but since it is not part of the standard plugin repository many are just not aware of it's existence.

**Somebody on the team** please add PR #5846 onto the [Kanban board](https://github.com/orgs/jellyfin/projects/55/views/2) and get it merged for the next release. It's time to end the confusion.

**Comment by colearrington** [comment]
For anyone who is still paying attention to this issue, the jellyfin-plugin-tmdb-trailers project (https://github.com/crobibero/jellyfin-plugin-tmdb-trailers) has been abandoned due to changes made by YouTube

**Comment by Xsphyre** [comment]
Seems absurd to me that it doesn't just pull all the trailers you have locally? (Movies with a movie_file-trailer or move_folder/trailers/file). Am I crazy? Why does it even need a plugin in the first place...
